homeassistant/components/binary_sensor/deconz.py

```diff
@@ -107,6 +107,8 @@ def device_state_attributes(self):
         attr = {}
         if self._sensor.battery:
             attr[ATTR_BATTERY_LEVEL] = self._sensor.battery
+        if self._sensor.on is not None:
+            attr[ATTR_ON] = self._sensor.on
```

Comment:
How is `is_on` different from `is_tripped` and `reachable` ? Shouldn't `on` just impact the `available` property?

Reply:
I was contemplating whether or not. But I decided to go with this way. The on is if the device should report status or not

Reply:
Would it be preferable if I called it enabled or keep with how zigbee names it?

Reply:
Yes, we should try to stick to how the source names things.